src/styles/learn.scss

```diff
@@ -181,7 +180,7 @@
       overflow: visible;
       overflow-y: scroll;
       position: fixed;
-      top: var(--nav-height);
+      top: +var(--nav-height);
```

Comment:
I don't understand the significance of `+` in here. Maybe it was a typo?

Reply:
Actually, I checked for this and this syntax doubles the value of the var that we're using but this might not be compatible with every browser so I have changed this to calc(2*var). Also, I have updated the snapshots as you mentioned.

Regards

Reply:
@kunaljain0212 I think you're missing a newline at the end of your package-lock.json, causing the diff that's still present.

Reply:
Also @kunaljain0212, I've added one more sub-issue in the original issue. Can you please take a look at that.

Thanks